lighthouse-core/report/html/renderer/details-renderer.js

```diff
@@ -304,16 +304,45 @@ class DetailsRenderer {
     }
 
     return tableLike.headings.map(heading => {
+      let multi;
+      if (heading.multi) {
+        multi = {
+          key: heading.multi.key,
+          valueType: heading.multi.itemType,
+          displayUnit: heading.displayUnit,
+          granularity: heading.granularity,
+        };
+      }
+
       return {
         key: heading.key,
         label: heading.text,
         valueType: heading.itemType,
+        multi,
         displayUnit: heading.displayUnit,
         granularity: heading.granularity,
       };
     });
   }
 
+  /**
+   * @param {LH.Audit.Details.TableItem} row
+   * @param {LH.Audit.Details.OpportunityColumnHeading} heading
+   * @return {Element?}
+   */
+  _renderMultiValue(row, heading) {
+    const values = row[heading.key];
+    if (!Array.isArray(values)) return null;
```

Comment:
this is always unexpected, right? any recovery options we have?

Reply:
this is mostly for ts. perhaps it should `console.error` something.

I suppose we could wrap a scalar value in an array of one item - but I would rather be strict about what we accept here. If the heading key was misconfigured, the audit developer will surely notice before publishing (otherwise, the data won't ever show!) Same for if the row value is scalar instead of the expected array. Then there's the case where it, by some ungodly bug, is sometimes scalar, sometimes an array - I bet that won't ever happen, so I think we shouldn't bother with a fallback.